### ISSUE TITLE
v2.0 does not respect --tab-width flag

### ISSUE THREAD

**Issue by shadowhand** [squizlabs/PHP_CodeSniffer#340]
We use PSR2, but with tabs instead of spaces. After upgrading to 
PHPCS 2.0, the `--tab-width=4` setting no longer works.

Reproduce by saving the following file as `Widget.php` (make sure tabs are preserved as tabs!) and running with `phpcs --standard=PSR2 --tab-width=4 Widget.php`.

``` php
<?php

namespace Acme;

class Widget
{
    /**
     * @return boolean
     */
    public function shouldBeNoError()
    {
        return true;
    }
}
```
## Expected result:

(no output, no errors)
## Actual result:

```
FILE: Widget.php
--------------------------------------------------------------------------
------
FOUND 7 ERRORS AFFECTING 7 LINES
--------------------------------------------------------------------------
------
  7 | ERROR | [x] Spaces must be used to indent lines; tabs are 
not allowed
  8 | ERROR | [x] Spaces must be used to indent lines; tabs are 
not allowed
  9 | ERROR | [x] Spaces must be used to indent lines; tabs are 
not allowed
 10 | ERROR | [x] Spaces must be used to indent lines; tabs are 
not allowed
 11 | ERROR | [x] Spaces must be used to indent lines; tabs are 
not allowed
 12 | ERROR | [x] Spaces must be used to indent lines; tabs are 
not allowed
 13 | ERROR | [x] Spaces must be used to indent lines; tabs are 
not allowed
--------------------------------------------------------------------------
------
PHPCBF CAN FIX THE 7 MARKED SNIFF VIOLATIONS 
AUTOMATICALLY
--------------------------------------------------------------------------
------

Time: 19ms; Memory: 3Mb
```
